### ISSUE TITLE
--local-folder to S3 bucket or Google Drive

### ISSUE THREAD

**Issue by viralkavish** [jay0lee/got-your-back#230]
I have bee using this for a while now and it is fantastic tool but is there a way to forward the --local-folder parameter to be more dynamic? In terms of S3 bucket or put the exported email folder to Google Drive? 

Also, is there any encryption can be done while taking backup? I understand we can do it after we download but it would be better if this can be added as a functionality. 

Thank You,

**Comment by jay0lee** [comment]
There are existing tools for syncing / backing up local files to Cloud storage, I'd suggest using those. Will keep this around as a possible enhancement but I don't see this as likely to happen anytime soon.

**Comment by viralkavish** [comment]
Thank you for the input! The idea is not to save anything locally and initiate everything on the cloud with encryption in place so there is no risk when we archive this for the future.

**Comment by jay0lee** [comment]
If that's the concern you'd be best served by leaving the data in place and using Google Vault for archiving/discovery of the mailbox.

**Comment by viralkavish** [comment]
Yes, though of that but we have a lot of users which we would like to preserve their inbox just for security measures but cost just adds up since we have to keep their account active or suspended. We cannot delete them, if we do delete them to re-purpose the license then data will be deleted from the vault as well. So, it eventually kills the purpose of having vault.

**Comment by jay0lee** [comment]
Take a look at the archived user aku:

https://support.google.com/a/answer/9048836

**Comment by bmenking-wng** [comment]
This has been implemented in another fork, if you want that functionality.  https://github.com/bmenking-wng/got-your-back

**Comment by no-decaf** [comment]
@viralkavish I backed up my emails to S3 by mounting a bucket to a local folder with [s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse)

**Comment by bterps** [comment]
you could also look at [rclone](https://github.com/rclone/rclone) as that can link with basically any cloud service
